frontend/src/helpers/validation.js

```diff
@@ -0,0 +1,11 @@
+import { helpers } from 'vuelidate/lib/validators';
+
+export const slackNickValidator = helpers.regex(
+  'slackNickValidator',
+  /^[0-9A-Za-ząćęłńóśźżĄĆĘŁŃÓŚŹŻ,.';\-_/()[\]{}]*$/
+);
+
+export const gitHubUsernameValidator = helpers.regex(
+  'gitHubUsernameValidator',
+  /^[0-9A-Za-z][0-9A-Za-z-]*[0-9A-Za-z]$/
```

Comment:
Whenever You want match the character - literally with regex, it is safer to escape it and use "\-" instead of "-".

Reply:
@stanislawK I have removed it as it causes lint failure: `error: Unnecessary escape character: \- (no-useless-escape) at src/helpers/validation.js:10:26`